File: Data Structures & Algorithms/two-integer-sum/submission-6.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int i = 0;
        

        while(i < nums.length) {
            int j = i + 1;
            while(j < nums.length){
                int compliment = target - nums[j];
                if(nums[i] == compliment){
                    return new int[] {i, j};
                }
                j +=1;  
            }
            i+=1;
        }

        return new int[]{};
        
    }
}

File: Data Structures & Algorithms/binary-search/submission-6.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number}
     */
    search(nums, target) {
        let res = -1;
        let low = 0;
        let high = nums.length - 1;

        while(low<=high){
            let mid = Math.floor((low+high)/2);

            if(nums[mid] < target){
                low = mid + 1
            } else if(nums[mid] > target){
                high = mid - 1
            } else {
                return mid
            }
        }

        return -1
    }
}